test/native/integration-test-helpers/add-block.js

```diff
@@ -11,11 +11,16 @@ import { waitFor } from './wait-for';
 /**
  * Adds a block via the block picker.
  *
- * @param {import('@testing-library/react-native').RenderAPI} screen                 A Testing Library screen.
- * @param {string}                                            blockName              Name of the block to be inserted as shown in the block picker.
- * @param {boolean}                                           [isPickerOpened=false] isPickerOpened Option to skip opening the inserter picker.
+ * @param {import('@testing-library/react-native').RenderAPI} screen                   A Testing Library screen.
+ * @param {string}                                            blockName                Name of the block to be inserted as shown in the block picker.
+ * @param {Object}                                            options                  Configuration options for adding a block.
+ * @param {boolean}                                           [options.isPickerOpened] Option to skip openning the inserter picker.
```

Comment:
I noticed a typo in the description of `isPickerOpened`:
```suggestion
 * @param {boolean}                                           [options.isPickerOpened] Option to skip opening the inserter picker.
```

Reply:
Nice catch! I think I needed another ☕ 😅